### PR TITLE
Fixed typos, simplified sentences

### PR DESCRIPTION
**Pull request by AjaxSolutions**
Thanks for taking the effort to create a PR!

If you are creating an extensive PR, you might want to open an issue with your idea first, so that you don't put a lot of effort in an PR that wouldn't be accepted. Please prepend pull requests with `WIP: ` if they are not yet finished
PR checklist:

* [ ] Added unit tests
* [ ] Updated changelog
* [ ] Updated docs (either in the description of this PR as markdown, or as separate PR on the `gh-pages` branch. Please refer to this PR). For new functionality, at least [API.md](https://github.com/mobxjs/mobx/blob/gh-pages/docs/refguide/api.md) should be updated
* [ ] Added typescript typings
* [ ] Verified that there is no significant performance drop (`npm run perf`)

Feel free to ask help with any of these boxes!

The above process doesn't apply to doc updates etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -237,53 +236,53 @@ Use them to your advantage; they will help you to structure your code better and
 
 ## MobX: Simple and scalable
 
-MobX is one of the least obtrusive libraries you can use for state management. That makes the `MobX` approach not just simple, but very scalable as well:
+MobX is a simple, very scaleable and obtrusive state management library.
```
Comment (mattruby) [inline README.md]: Unobtrusive

File: README.md [hunk 2]
```diff
@@ -96,12 +95,12 @@ class Todo {
...
 
 Using `observable` is like turning a property of an object into a spreadsheet cell.
-But unlike spreadsheets, these values can be not only primitive values, but also references, objects and arrays.
+But, unlike spreadsheets, these values can be not only primitive values, but also references, objects and arrays.
 
 If your environment doesn't support decorator syntax, don't worry.
 You can read [here](http://mobxjs.github.io/mobx/best/decorators.html) about how to set them up.
 Or you can skip them altoghether, as MobX can be used fine without decorator _syntax_, by leveraging the _decorate_ utility.
-Many MobX users do prefer the decorator syntax though, as it is slightly more concise.
+Many MobX users prefer the slightly more concise decorator syntax, though.
```
Comment (mattruby) [inline README.md]: Remove though

File: README.md [hunk 3]
```diff
@@ -237,53 +236,53 @@ Use them to your advantage; they will help you to structure your code better and
...
 
 ### Referential integrity is guaranteed
 
-Since data doesn't need to be normalized, and MobX automatically tracks the relations between state and derivations, you get referential integrity for free. Rendering something that is accessed through three levels of indirection?
+Since data doesn't need to be normalized and MobX automatically tracks the relations between state and derivations, you get referential integrity for free.
 
-No problem, MobX will track them and re-render whenever one of the references changes. As a result staleness bugs are a thing of the past. As a programmer you might forget that changing some data might influence a seemingly unrelated component in a corner case. MobX won't forget.
+Rendering something that is accessed through three levels of indirection? No problem. MobX will track them and re-render whenever one of the references changes. As a result, staleness bugs are avoided. As a programmer, you might forget that changing some data might influence a seemingly unrelated component, but MobX won't forget.
```
Comment (mattruby) [inline README.md]: Replace avoided with eliminated

File: README.md (hunk 1 above)
Comment (AjaxSolutions) [reply README.md]: Good eye :)

**Comment by mattruby** [comment]
@AjaxSolutions thanks for all your work on this!

**Pull request merged**